LICENSE.md

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2019, University of Dundee
```

Comment:
Shouldn't the name of the licence be included?

Reply:
I don't think so. At least neither https://opensource.org/licenses/BSD-2-Clause nor https://github.com/licensee/licensee (which is the library used by GitHub for detection) include it. Do you have an example where the name is included?

Reply:
I don't, but just looking at the licence I couldn't tell whether it was a standard opensource licence, or a custom one which needs to be read carefully.

Reply:
Understood. For reference, I simply copied/pasted a BSD-2 license from one of our repos and updated the header line. My assumption is that any custom changes to the license text will be flagged by GitHub not associating BSD-2 to the repository on merging.

Reply:
Update: to quickly test the above, I fast-forwarded the `master` branch of my fork to the HEAD of this PR and https://github.com/sbesson/ansible-roles correctly detects the repo as `BSD-2`